src/ResourceManager/Network/Commands.Network/help/Get-AzureRmVirtualHub.md

```diff
@@ -0,0 +1,130 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.network/get-azurermvirtualhub
+schema: 2.0.0
+---
+
+# Get-AzureRmVirtualHub
+
+## SYNOPSIS
+Gets an Azure VirtualHub by Name and ResourceGroupName or lists all Virtual Hubs by ResourceGroupName/Subscription.
+
+## SYNTAX
+
+### ListByResourceGroupName (Default)
+```
+Get-AzureRmVirtualHub -ResourceGroupName <String> [-Name <String>] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+### ListBySubscriptionId
+```
+Get-AzureRmVirtualHub [-ResourceGroupName <String>] [-Name <String>] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Gets an Azure VirtualHub by Name and ResourceGroupName or lists all Virtual Hubs by ResourceGroupName/Subscription.
+
+## EXAMPLES
+
+### Example 1
+
+```powershell
+PS C:\> New-AzureRmResourceGroup -Location "West US" -Name "testRG"
+PS C:\> $virtualWan = New-AzureRmVirtualWan -ResourceGroupName "testRG" -Name "myVirtualWAN" -Location "West US"
+PS C:\> New-AzureRmVirtualHub -VirtualWan $virtualWan -ResourceGroupName "testRG" -Name "westushub" -AddressPrefix "10.0.1.0/24"
+PS C:\> Get-AzureRmVirtualHub -ResourceGroupName "testRG" -Name "westushub"
+
+Output:
```

Comment:
don't need this "Output:" line.  Please remove everywhere.

Reply:
Fixed.